samples/subsys/usb/cdc_acm/README.rst

```diff
@@ -0,0 +1,63 @@
+.. _usb_cdc_acm:
+
+USB CDC ACM Sample Application
+####################################
+
+Overview
+********
+
+Sample app for USB CDC ACM driver. The received data is echoed back
+to the serial port provided by Zephyr CDC ACM driver.
+
+Requirements
+************
+
+This project requires an USB device driver,
+which is available for the :ref:`arduino_101` and :ref:`frdm_k64f` boards.
```

Comment:
yes, but, I've just seen that the default board for this sample is quark_se_c1000_devboard.